llm/yamls/mosaic_gpt/3b.yaml

```diff
@@ -73,7 +68,7 @@ scheduler:
 
 optimizer:
   name: decoupled_adamw
-  lr: 2.0e-4
+  lr: 1.6e-4
```

Comment:
@vchiley RIP

Reply:
I'm not sure if you were copying this exactly for the Legal LLM work but if so the LR may have been different from the GPT3 original hparams.